cmd/internal/cli/build.go

```diff
@@ -506,34 +506,35 @@ func definitionFromSpec(spec string) (types.Definition, error) {
 	return def, nil
 }
 
-// makeDockerCredentials creates an *ocitypes.DockerAuthConfig to use for
-// OCI/Docker registry operation configuration. Note that if we don't have a
-// username or password set it will return a nil pointer, as containers/image
-// requires this to fall back to .docker/config based authentication.
-func makeDockerCredentials(cmd *cobra.Command) (authConf *ocitypes.DockerAuthConfig, err error) {
+// makeOCICredentials creates an *authn.AuthConfig to use for OCI/Docker
+// registry operation configuration. Note that if we don't have a username or
```

Comment:
Given that this function calls out to interactive login under certain circumstances, would it make sense to rewrite the part of the comment beginning with "Note that if" (probably written before interactive login was even implemented...)?